### PR TITLE
Finally, finally, close #12164

### LINKED ISSUE #12164: Add an event-state diagram for containers
In the course of a pull request a contributor provided an event-state diagram to illustrate his technical point: (https://github.com/docker/docker/pull/11827#issuecomment-90760920)

The example he produced looks like this:

image

The diagram includes **proposed** states but eliminating those would produce a pretty sweet diagram for the docs.  We could use the diagram in a container states page.

### PR DESCRIPTION
**Pull request by moxiegirl**
Matt et. al. my deepest apologies for waiting till the last minute on this one.  Let me know if it needs an update. I'll make it a priority.

Signed-off-by: Mary Anthony mary@docker.com

### CODE REVIEW

**Comment by thaJeztah** [comment]
Thanks! LGTM

File: docs/sources/reference/api/docker_remote_api.md [hunk 1]
```diff
@@ -38,6 +38,22 @@ Calling `/info` is the same as calling
 You can still call an old version of the API using
 `/v1.18/info`.
 
+## Docker Events
+
+The following diagram depicts the container states accessible through the API.
+
+![States](../images/event_state.png)
+
+Some container-related are not affected by container state, so they are not included in this diagram. They are:
```
Comment (thaJeztah) [inline docs/sources/reference/api/docker_remote_api.md]: missing word; "Some container-related *states*"

Comment (thaJeztah) [inline docs/sources/reference/api/docker_remote_api.md]: Sorry "events" probably :)

**Comment by thaJeztah** [comment]
@moxiegirl spotted one small typo, but otherwise LGTM :)

**Comment by moxiegirl** [comment]
@thaJeztah done.  ping @docker/docs-owners

**Comment by moxiegirl** [comment]
Author LGTM

**Pull request merged**